### PR TITLE
Hcal noise filters backport

### PR DESCRIPTION
**Pull request by igv4321**
Backporting PR #8685

Compared to PR #8685, there is a minor difference in the config files -- the "HBHENoiseFilterResultRun2Loose" flag is made default only for 25 ns Run 2 rather than for the whole Run 2. For 50 ns Run 2 we still use "HBHENoiseFilterResultRun1".

### CODE REVIEW

**Comment by slava77** [comment]
@cmsbuild please test

File: SLHCUpgradeSimulations/Configuration/python/postLS1Customs.py [hunk 1]
```diff
@@ -49,6 +49,8 @@ def customisePostLS1(process):
     # 25ns specific customisation
     if hasattr(process,'digitisation_step'):
         process = customise_Digi_25ns(process)
+    if hasattr(process,'dqmoffline_step'):
+        process = customise_DQM_25ns(process)
```
Comment (slava77) [inline SLHCUpgradeSimulations/Configuration/python/postLS1Customs.py]: why not do the same in 75X?

Comment (igv4321) [inline SLHCUpgradeSimulations/Configuration/python/postLS1Customs.py]: Yes, will eventually do the same in 75X. The proper Run 2 config wasn't obvious when the implementation was originally discussed.

**Comment by slava77** [comment]
@cmsbuild @nclopezo regarding https://github.com/cms-sw/cmssw/pull/8805#issuecomment-94633340
David, is the check of step1_dasquery done only for 140.53?
I see that this file was different in 1000.0 and 1001.0 workflows as well

**Comment by nclopezo** [comment]
The check is supposed to be done for every workflow. I will check what happened.

**Comment by slava77** [comment]
+1

for https://github.com/cms-sw/cmssw/pull/8805	7be30ee
the backport is essentially consistent with the change that went to 75X.

- visual comparison of the code differences shows the same changes as in https://github.com/cms-sw/cmssw/pull/8685	d0d4741, except for the corresponding choice of the default value in HBHENoiseFilterResultProducer (discussed above); OK
- the comparisons in jenkins show differences in expected places: 
    - only noiseInfo products change in stored data products
    - DQM is consistent with that and has changes only in  /HcalNoiseRatesD/\|/NoiseMonitor_Hcal/\|/NoiseRatesV/ folders.

**Comment by nclopezo** [comment]
Regarding https://github.com/cms-sw/cmssw/pull/8805#issuecomment-94779751, I fixed it, it should be ok now.

**Comment by civanch** [comment]
+1

**Comment by davidlange6** [comment]
+1

**Pull request merged**